lib/components/Information/Avatar/index.tsx

```diff
@@ -1 +1,23 @@
-export * from "@/ui/avatar"
+import {
+  Avatar as AvatarComponent,
+  AvatarFallback,
+  AvatarImage,
+} from "@/ui/avatar"
+import { forwardRef } from "react"
+
+interface AvatarType {
+  alt: string
+  src?: string
+  size?: "medium" | "small" | "xsmall" | "large" | "xlarge" | "xxlarge"
```

Comment:
If you want to delegate to the sizes that `AvatarComponent` allows, you can always do:

```suggestion
  size?: ComponentProps<typeof AvatarComponent>["size"]
```

Reply:
done